rxandroid/src/main/java/rx/android/app/AppObservable.java

```diff
@@ -79,13 +78,30 @@ public Boolean call(android.support.v4.app.Fragment fragment) {
         return source.observeOn(mainThread()).lift(new OperatorConditionalBinding<T, Activity>(activity, ACTIVITY_VALIDATOR));
     }
 
+    @Deprecated
+    public static <T> Observable<T> bindFragment(Object fragment, Observable<T> source) {
```

Comment:
Just delete. The overloads will resolve on upgrade for anyone using this correctly. And if you weren't then we just turned your runtime failure to a compile time failure.